host_applications/linux/apps/raspicam/RaspiVid.c

```diff
@@ -2295,15 +2295,46 @@ static int wait_for_next_change(RASPIVID_STATE *state)
       char ch;
 
       if (state->verbose)
-         fprintf(stderr, "Press Enter to %s, X then ENTER to exit\n", state->bCapturing ? "pause" : "capture");
+         fprintf(stderr, "Press Enter to %s, X then ENTER to exit, [i,o,r] then ENTER to change zoom\n", state->bCapturing ? "pause" : "capture");
```

Comment:
getchar() is line buffered by default, so you do need enter to see anything.
You can get around this with tcsetattr, e.g.
https://github.com/popcornmix/omxplayer/blob/master/Keyboard.cpp#L28
but I wouldn't recommend that. If you don't put it back it breaks the console input, so you need signal handlers for unexpected exit paths (e.g. crashes) so it gets ugly.